integration-tests/testsetups/ocr.go

```diff
@@ -598,7 +601,47 @@ func (o *OCRSoakTest) testLoop(testDuration time.Duration, newValue int) {
 		} else {
 			reorgDelay = n.GethReorgConfig.DelayCreate.Duration
 		}
-		reorgCh = time.After(reorgDelay)
+		time.AfterFunc(reorgDelay, func() {
+			if !o.reorgHappened {
+				o.startGethBlockchainReorg(o.rpcNetwork, n.GethReorgConfig)
```

Comment:
curious: why `GethBlockchainReorg` and not `Anvil...`?

Reply:
@Tofel we use geth blockchain for rerorg scenarios by default